Source/MLX/Factory.swift

```diff
@@ -294,13 +294,43 @@ extension MLXArray {
             mlx_linspace(Double(start), Double(stop), count.int32, T.dtype.cmlxDtype, stream.ctx))
     }
 
+    /// Repeat an array along a specified axis.
+    ///
+    /// > Deprected in favor of the more consistently named ``repeated(_:count:axis:stream:)``
+    ///
+    /// ### See Also
+    /// - <doc:initialization>
+    /// - ``repeated(_:count:stream:)``
+    /// - ``full(_:values:stream:)``
+    @available(*, deprecated, renamed: "repeated(_:count:axis:stream:)")
+    static public func `repeat`(
```

Comment:
This will still build and work but you will get a warning and a code-fix that will change it to `repeated()`